components/Layout/BackButton/BackButton.tsx

```diff
@@ -1,7 +1,7 @@
 import Router from 'next/router';
 
 const BackButton = (): React.ReactElement => (
-  <a className="govuk-back-link" role="button" onClick={() => Router.back()}>
+  <a className="govuk-back-link" href="#" onClick={() => Router.back()}>
```

Comment:
Not a massive fan of `href="#"` but if the accessibility test didn't complain about it I guess is fine